### ISSUE TITLE
Provide requirements.txt

### ISSUE THREAD

**Issue by tuukka** [urbangrammarai/urbangrammarai.github.io#13]
Could you list the dependencies in requirements.txt so that the tesselation notebook could work on mybinder.org?

**Comment by martinfleis** [comment]
Thanks for a suggestion! The whole research project runs in `gds_env` docker container (https://urbangrammarai.github.io/data_processing/infrastructure/platform.html) so I'll try to find a way how to hook that into binder (@darribas ideas?).

Alternatively, we would have to create conda environment for binder.

**Comment by darribas** [comment]
I'd suggest to run Binder off the `gds_py:5.0` container. See an example here for the `gds_course`:

https://github.com/darribas/gds_course/blob/master/Dockerfile

Happy to assist further if needed.

**Comment by tuukka** [comment]
Thanks for your suggestions! With the Dockerfile at PR #14, I was able to run the notebook in Binder, but there were multiple errors regarding imports:
```
NameError: name 'ox' is not defined
AttributeError: module 'osmnx' has no attribute 'geometries'
NameError: name 'Point' is not defined
```

I have fixes at PR #15. At this point, the other examples work but the LA and BOL fail with separate errors:
https://nbviewer.jupyter.org/github/tuukka/urbangrammarai.github.io/blob/fix%2fadd-missing-imports/src/notebooks/mapchallenge/tessellation.ipynb

```
ValueError: Length of values (3797) does not match length of index (4000)
ValueError: Shape of passed values is (808, 31), indices imply (803, 31)
```

Also, can you suggest a way to view the resulting images in Binder?

**Comment by martinfleis** [comment]
I have now pushed a fix for all you found and prepared minimal environment for binder.

`AttributeError: module 'osmnx' has no attribute 'geometries'` this is caused by version mismatch, notebook uses newer build of `gds_py` which has recent osmnx.

If you now open repo in mybinder, it should all work.

> Also, can you suggest a way to view the resulting images in Binder?

Yes, once your instance will open, change `/tree` to `/lab` which will open JupyterLab UI which allows you to see the figures.

Thanks a lot for your comments and suggestions!
